docs/reference/monitor.md

```diff
@@ -48,6 +48,15 @@ In addition, lakeFS exposes the following metrics to help monitor your deploymen
 | dynamo_request_duration_seconds  | Time spent doing DynamoDB requests                          | **operation**: DynamoDB operation name
```

Comment:
We need to update the sizing guide metrics section for both dynamodb and postgres

Reply:
The specific metrics added for pxpool currently helps monitoring connection usage. It can help sizing, but it is not like the usage metrics we have for dynamodb.
Postgres performance monitoring can be does externally, not like the internal pool we monitor here.